doc/UsersGuide/source/tables/fv3diagtable.csv

```diff
@@ -0,0 +1,421 @@
+facwf,Fractional coverage with weak cos(z) dependency
+ffhh,FH parameter from planetary boundary layer scheme
+ffmm,FM parameter from planetary boundary layer scheme
+uustar,U-uStar surface frictional wind
+slope,Surface slope type
+fice,Surface ice concentration (ice=1; no ice=0)
+hice,Sea ice thickness (icetk in gfs_data)
+snoalb,Maximum snow albedo in fraction
+shdmax,Maximum fractional coverage of green vegetation
+shdmin,Minimum fractional coverage of green vegetation
+snowd,Surface snow depth
+sbsno,Instantaneous sublimation (evaporation from snow)
+evbs,Instantaneous direct evaporation over land
+evcw,Instantaneous canopy evaporation
+trans,Instantaneous transpiration
+sfalb,Surface albedo over land
+rhofr,Density of frozen precipitation
+snowfall_acc_land,Total accumulated frozen precipitation over land
+acsnow_land,Total accumulated SWE (snow water equivalent) of frozen precipitation over land
+snowmt_land,Accumulated snow melt over land
+snowfall_acc_ice,Total accumulated frozen precipitation over ice
+acsnow_ice,Total accumulated SWE (snow water equivalent) of frozen precipitaton over ice
+snowmt_ice,Accumulated snow melt over ice
+crain,Instantaneous categorical rain
+stype,Soil type in integer 1-9
+scolor,Soil color in integer 1-20
+lfrac,Land fraction
+q2m,2 meter specific humidity
+t2m,2 meter temperature
+tsfc,Surface temperature
+usfco,Surface zonal current
+vsfco,Surface meridional current
+tiice,Internal ice temperature layer
+tg3,Deep soil temperature
+tisfc,Surface temperature over ice fraction
+tprcp,Total time-step precipitation
+vtype,Vegetation type in integer
+weasd,Surface snow water equivalent
+weasdi,Surface snow water equivalent over ice
+snodi,Snow depth over ice
+hgtsfc,Surface geopotential height
+slmsksfc,"Sea-land-ice mask (0-sea, 1-land, 2-ice)"
+zorlsfc,Surface roughness
+vfracsfc,Vegetation fraction
+wetness,Soil moisture availability in top soil layer
+nirbmdi,Surface near IR beam shortwave downward flux
+nirdfdi,Surface near IR differential shortwave downward flux
+visbmdi,Surface uv+visible beam shortwave downward flux
+visdfdi,Surface uv+visible differential shortwave downward flux
+xlaixy,Leaf area index
+vfrac,Fraction of vegetation category
+sfrac,Fraction of soil category
+slc,Liquid soil moisture
+soill,Liquid soil moisture
+soilw,Volumetric soil moisture
+soilt,Soil temperature
+tref,NSST reference or foundation temperature
+z_c,NSST sub-layer cooling thickness
+c_0,NSST coefficient1 to calculate d(tz)/d(ts)
+c_d,NSST coefficient2 to calculate d(tz)/d(ts)
+w_0,NSST coefficient3 to calculate d(tz)/d(ts)
+w_d,NSST coefficient4 to calculate d(tz)/d(ts)
+xt,NSST heat content in diurnal thermocline layer
+xs,NSST salinity content in diurnal thermocline layer
+xu,NSST u-current content in diurnal thermocline layer
+xv,NSST v-current content in diurnal thermocline layer
+xz,NSST diurnal thermocline layer thickness
+zm,NSST mixed layer thickness
+xtts,NSST d(xt)/d(ts)
+xzts,NSST d(xt)/d(ts)

```

Comment:
```suggestion
xzts,NSST d(xz)/d(ts)
```
I think the `desc` field is a typo/copy-paste error because the old CCPP docs describe xzts as d(xz)/d(ts): https://dtcenter.ucar.edu/gmtb/users/ccpp/docs/sci_doc/group__gfs__nst__main.html#arg_table_sfc_nst_run
In general, the variable name seems to describe the values in parentheses.